src/test/lib/optimizer/strategy/between_composition_rule_test.cpp

```diff
@@ -443,9 +468,17 @@ TEST_F(BetweenCompositionRuleTest, HandleMultipleEqualExpressions) {
 
   const auto expected_lqp = _lqp->deep_copy();
 
-  _apply_rule(_rule, _lqp);
+  _apply_rule(_rule, _lqp, _optimization_context);
 
+  EXPECT_TRUE(_optimization_context.is_cacheable());
   EXPECT_LQP_EQ(_lqp, expected_lqp);
 }
 
+TEST_F(BetweenCompositionRuleTest, CheckCacheability) {
+  auto input_lqp = std::dynamic_pointer_cast<AbstractLQPNode>(
+      PredicateNode::make(equals_(_a_a, 100), PredicateNode::make(equals_(_a_b, 100), _node_a)));
+  _apply_rule(_rule, input_lqp, _optimization_context);
+  EXPECT_TRUE(_optimization_context.is_cacheable());
```

Comment:
pick here: You can use `StrategyBaseTest`'s `_lqp` member. Please also have a look at the way we usually format LQPs we create in tests, e.g.: 
https://github.com/hyrise/hyrise/blob/fbc8b1fd98fe5a0772162f2b4338f5a1755f7cb4/src/test/lib/sql/sql_translator_test.cpp#L1398-L1406